.pre-commit-config.yaml

```diff
@@ -32,3 +32,9 @@ repos:
   rev: v0.982
   hooks:
   - id: mypy
+    additional_dependencies: [types-pytz==2022.4.0.0,
+                              types-ujson==5.5.0,
+                              types-python-dateutil==2.8.19,
+                              types-requests==2.28.11.2,
+                              types-attrs==19.1.0,
+                              types-PyYAML==6.0.12]
```

Comment:
These are actually duplicates of values from requirements.in which is not great.

Reply:
looks like we need to specify the additional_dependencies...